src/App.js

```diff
@@ -9,7 +10,12 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <>
+    <h1>
+      React Article
+    </h1>
```

Comment:
Page usually should have only on `h1` element, so this header can be safely deleted.